### DISCUSSION TITLE
Error Encountered While Installing Supabase Dependencies

### DISCUSSION THREAD

**Discussion by shishiro26** [supabase/supabase#18529] [Contribute to Supabase]
I am having trouble installing the dependencies in my local repository. I followed the documentation and attempted to use npm install, but I encountered the following error:
![image](https://github.com/supabase/supabase/assets/113853868/4586d086-1ee2-42cb-8d80-668ea2656c16)

I faced the same error when I tried to clone the repository on my friend's laptop. It seems to be a persistent issue. Can you please help me resolve this problem?"

**Comment by SubhoYadav** [comment]
I am also experiencing the same issue even though my node and npm versions are compatible with the documentation. Is your issue resolved ?
